app/api/exhibitors.py

```diff
@@ -30,13 +30,24 @@ def query(self, view_kwargs):
         query_ = event_query(query_, view_kwargs)
         return query_
 
+    def after_create_object(self, exhibitor, data, view_kwargs):
+        if data.get('banner_url'):
+            start_image_resizing_tasks(exhibitor, data['banner_url'])
+
     view_kwargs = True
-    methods = ['GET']
+    methods = ['GET', 'POST']
     schema = ExhibitorSchema
-    data_layer = {'session': db.session, 'model': Exhibitor, 'methods': {'query': query}}
+    data_layer = {
+        'session': db.session,
+        'model': Exhibitor,
+        'methods': {'after_create_object': after_create_object, 'query': query},
+    }
 
 
 class ExhibitorDetail(ResourceDetail):
+    def before_update_object(self, exhibitor, data, view_kwargs):
+        if data.get('banner_url'):
```

Comment:
This will resize the image everytime a PATCH request is sent, regardless of banner URL being changed or not